rmgpy/test_data/testing_database/solvation/groups/abraham.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+name = "Abraham Solute Descriptors"
+shortDesc = u""
+longDesc = u"""
+
+"""
+entry(
+    index = -3,
```

Comment:
The solvation database was pulled from RMG-database. [It](https://github.com/ReactionMechanismGenerator/RMG-database/commit/2479933de81dc6d14209368043bd6394964c2e5a#diff-8d02a65677f4c2ecf8810f77ebfde7cb) looks like it's been here for years. The reasoning seemed to be to stick things at the top and add a previous number.